### PR TITLE
Board review

### PR DESCRIPTION
**Pull request by duardoqueiroz**
# Proposed Changes
- implement board review system

# Aditional info 
- need to configure editing review

# Screenshots
> # Updated
image
image

### CODE REVIEW

**Review by leticiabsduarte — CHANGES_REQUESTED** [review]
- Let only one number after dot, and change it to a comma
estrelinhas

- Each commentary has your account with your different icon. Fix this error with the icons profile
resenharesenha

- I believe you have access to the Figma of this site, always make it exactly what Stanislav did
original russo
original russo 2

File: app/Models/Board.ts [hunk 1]
```diff
@@ -88,10 +88,12 @@ export default class Board extends BaseModel {
...
     return this.reviews.length > 0
       ? this.reviews
-          .map((review) => review.rating)
+          .map((review) => {
+            return review.rating
+          })
           .reduce((count, el) => {
-            return count + el
-          }) / this.reviewNumber
+            return parseFloat(count.toString()) + parseFloat(el.toString())
```
Comment (Manuel-Antunes) [inline app/Models/Board.ts:95]: If you wan't to use this field, remove unused computed `reviewNumber`

File: app/Models/Event.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ export default class Event extends BaseModel {
       ? this.reviews
           .map((review) => review.rating)
           .reduce((count, el) => {
-            return count + el
+            return parseInt(count.toString()) + parseInt(el.toString())
```
Comment (Manuel-Antunes) [inline app/Models/Event.ts:78]:
```suggestion
            return parseInt(count.toString(), 10) + parseInt(el.toString(), 10)
```
add radix to parseInt function

File: app/Models/User.ts [hunk 3]
```diff
@@ -75,7 +75,7 @@ export default class User extends BaseModel {
       ? this.reviews
           .map((review) => review.rating)
           .reduce((count, el) => {
-            return count + el
+            return parseInt(count.toString()) + parseInt(el.toString())
```
Comment (Manuel-Antunes) [inline app/Models/User.ts:78]:
```suggestion
            return parseInt(count.toString(), 10) + parseInt(el.toString(), 10)
```
add radix

File: resources/views/components/hexagonal-photo.edge [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<div class="hex flex relative" style="-webkit-clip-path: polygon(50% 1%, 95% 25%, 95% 75% ,50% 99%, 5% 75%, 5% 25%);
```
Comment (Manuel-Antunes) [inline resources/views/components/hexagonal-photo.edge:1]: Turn this style tag into a custom class in tailwind called `hexagon`

**Review by Manuel-Antunes — CHANGES_REQUESTED** [review]
Check my issues

**Review by leticiabsduarte — CHANGES_REQUESTED** [review]
- This is what happens when there are a big commentary and more than two reviews, fix it.

avaliação erro

avaliação erro 2
- The icons profile keeps on bug

- Add the half star in ratings, as figma

original russo 2

**Comment by duardoqueiroz** [comment]
> the bugs are already fixed. Please try to git pull the modifications.
image
#
image

**Comment by leticiabsduarte** [comment]
> > the bugs are already fixed. Please try to git pull the modifications. @leticiabsduarte
> > [#78 (review)](https://github.com/Manuel-Antunes/taverna-de-mordor-adonis/pull/78#pullrequestreview-871744983)
> > image
> 
> image

my bad, it's all perfect now =)

**Review by leticiabsduarte — APPROVED** [review]
good work!

**Review by Manuel-Antunes — APPROVED** [review]
Please, move hexagon class to tailwind config

**Comment by guilherme-gjv** [comment]
## Dudu asked me to merge it...

**Pull request merged**